resources/windows/wix/service-filter.xslt.erb

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<xsl:stylesheet version="1.0"
+ xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+ xmlns:wix="http://schemas.microsoft.com/wix/2006/wi">
+  <!-- https://ahordijk.wordpress.com/2013/03/26/automatically-add-references-and-content-to-the-wix-installer/ -->
+  <!-- http://www.chriskonces.com/using-xslt-with-heat-exe-wix-to-create-windows-service-installs/ -->
+  <xsl:output method="xml" indent="yes" />
+  <!--<xsl:strip-space elements="*"/>-->
+  <xsl:template match="@*|node()">
+    <xsl:copy>
+      <xsl:apply-templates select="@*|node()"/>
+    </xsl:copy>
+  </xsl:template>
+
+  <!-- Loop through the list of services and add filters for each component requiring Service type special treatment -->
+  <%- get_services.each do |service| -%>
+
+      <xsl:template match="wix:Component[wix:File[@Source='<%= service.options.first[:file][:source] %>"']]" />
+

```

Comment:
Is this extra white space necessary? Also, I'm not sure about the comment. Maybe something with a little more clarity? Like, why we have to filter these? Don't worry about the prefix. From the conversation this morning, this will change.